app/controllers/blacklisted_controller.rb

```diff
@@ -5,7 +5,8 @@ def is_blacklisted
 
     blacklist = Blacklist.find_by(
       user: User.find_by(api_token: api_token),
-      merchant_name: merchant_name
+      merchant_name: merchant_name,
+      blocked: true
     )
 
     blocked = true
```

Comment:
Is the additional logic in use here still necessary in the face of the updated query?